### ISSUE TITLE
normalize 方法中最后将未来预测均值和标准差沿着通道维度进行拼接部分为什么不用公式进行描述和解释

### ISSUE THREAD

**Issue by enjoy66613** [icantnamemyself/SAN#14]
outputs = torch.cat([outputs_mean, outputs_std], dim=-1)

**Comment by icantnamemyself** [comment]
感谢您对我们工作的关注。此处只是一种代码的写法，用于方便后续计算，见exp_main.py的57到64行
